### DISCUSSION TITLE
The prompt template for Yi-34b

### DISCUSSION THREAD

**Discussion by aisensiy** [ggml-org/llama.cpp#4533] [Q&A]
I am using yi-34b-chat.Q4_K_M.gguf with llama.cpp , but I have issues like this:

https://github.com/ggerganov/llama.cpp/assets/661860/5d4c9419-4754-4b37-b242-b40d51fcb234

Why the llm continue print out things after `<|im_end|>`. How can I config it to make it work like chatgpt.

**Comment by Jeximo** [comment]
> Why the llm continue print out things after <|im_end|>

Do you use `--chat-ml` parameter?
